src/main/scala/cromwell/engine/io/gcs/GoogleConfiguration.scala

```diff
@@ -12,6 +12,7 @@ import scalaz._
 sealed trait GoogleCromwellAuthMode
 final case class ServiceAccountMode(accountId: String, pemPath: String) extends GoogleCromwellAuthMode
 final case class UserMode(user: String, secretsFile: String, datastoreDir: String) extends GoogleCromwellAuthMode
+final case object ApplicationDefaultMode extends GoogleCromwellAuthMode
```

Comment:
Yet sadly some prominent scala trainers advocate adding it. Clearly they're not IntelliJ users :)